### DISCUSSION TITLE
ArewaDS FellowShip - Arewa Data Science Application is Open

### DISCUSSION THREAD

**Discussion by giscus** [EllaKaye/ellakaye-quarto#8] [General]
# ArewaDS FellowShip - Arewa Data Science Application is Open

Application for the ArewaDS Cohort2 is Open and closes on 8 September

https://arewadatascience.github.io/posts/Cohort2_Application/

**Comment by Mkalbani** [comment]
When are you starting?

**Reply by EllaKaye** [reply]
Hello. Your message for Arewa Data Science has come through to me by mistake. They based their website on my website and didn't update the giscus comments setup so it's still linked to my site. Please contact them directly at arewadatascience@gmail.com. Best wishes.

**Comment by Mkalbani** [comment]
When are you starting the program

**Comment by Mkalbani** [comment]
Thank you. I'll do that.
